pkg/monitor/cluster/clusterversions.go

```diff
@@ -51,13 +51,12 @@ func (mon *Monitor) emitClusterVersions(ctx context.Context) error {
 	mon.emitGauge("cluster.versions", 1, map[string]string{
 		"actualVersion":                        actualVersion,
 		"desiredVersion":                       desiredVersion(cv),
-		"provisionedByResourceProviderVersion": mon.oc.Properties.ProvisionedBy,                     // last successful Put or Patch
-		"resourceProviderVersion":              version.GitCommit,                                   // RP version currently running
-		"operatorVersion":                      operatorVersion,                                     // operator version in the cluster
-		"availableVersion":                     availableVersion(cv, version.UpgradeStreams),        // current available version for upgrade from stream
-		"availableRP":                          availableRP,                                         // current RP version available for document update, empty when none
-		"latestGaMinorVersion":                 version.DefaultInstallStream.Version.MinorVersion(), // Latest GA in ARO Minor version
```

Comment:
is there a reason we've dropped the `latestGaMinorVersion` metric?

Reply:
I couldn't find any evidence of its use, and `version.DefaultInstallStream` is strictly for local development only now.